static/monorail-2.0.yaml

```diff
@@ -221,6 +247,24 @@ definitions:
             type: object
         type: object
     type: object
+  relations.2.0_Relation:
+    description: A key value set of relations.
+    properties:
+      containedBy:
```

Comment:
Besides `containedBy` and `contains`, there are still other relations (see https://github.com/RackHD/on-core/blob/master/lib/common/constants.js#L256 )